templates/components/common/cart-preview.html

```diff
@@ -77,7 +77,7 @@ <h6 class="previewCartItem-name">
                 </div>
             {{/if}}
         {{else}}
-            <div class="previewCart-emptyBody">
+            <div role="alert" aria-live="polite" aria-atomic="false" class="previewCart-emptyBody">
```

Comment:
Why do we need to explicitly set aria-atiomic to false here if it is a default value?

Reply:
In other case message "Your cart is empty" is double announced by screen reader